lib/shared/client.ts

```diff
@@ -57,16 +66,18 @@ export class BaseClient {
         "Basic " + base64Encode(config.project_id + ":" + config.secret),
     };
 
+    const baseURL = config.base_url ? config.base_url : config.env;
```

Comment:
If the linter will let you get away with it 😎 
```suggestion
    const baseURL = config.base_url || config.env;
```